### PR TITLE
Add option to pass in openJ9 repo and branch

### PR DESCRIPTION
**Pull request by renfeiw**
- add openj9 repo and branch
- minor format change by running npm format

Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Review by sophia-guo — APPROVED** [review]
LGTM

File: action.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ inputs:
     description: 'openjdk Repo'
     required: false
     default: 'openjdk-tests:master'
+  openj9_repo:
+    description: 'openj9 Repo'
+    required: false
+    default: 'openj9:master'
```
Comment (llxia) [inline action.yml:27]: Should use the new repo? i.e., `eclipse-openj9:openj9`

Comment (llxia) [reply action.yml:27]: Talked with @renfeiw , this is used as a check. It is not used for constructing openj9 git repo url, so it does not need to be changed.

Comment (sophia-guo) [reply action.yml:27]: openj9 is the OpenJ9 main code base under eclipse-openj9 : https://github.com/eclipse-openj9/openj9. I think openj9 is a correct one. Also I believe `openj9_repo` is set as the same way as two other repos do, which only use default as a flag. We actually don't need the default value as we can check if the parameter is empty or not. I will open a separate issue to do that.

**Review by llxia — APPROVED** [review]

**Review by smlambert — APPROVED** [review]
LGTM

**Pull request merged**